README.md

```diff
@@ -16,9 +16,156 @@ devtools::install_github("MethodsConsultants/tibbletest")
+    ## The following variable names were cleaned with the mapping:
+    ## 
+    ## Original Name    Cleaned Name 
+    ## ---------------  -------------
+    ## sugar   factor   sugar_factor
+
+    ## 
+
+    ## [Step 3]: Adding variable labels
+    ## The following variable labels were specified with the mapping:
+    ## 
+    ## Variable       Label        
+    ## -------------  -------------
+    ## gender         Gender       
+    ## age            Age          
+    ## sugar_factor   Sugar Factor 
+    ## treat          Treat        
+    ## happiness      Happiness    
+    ## happy          Happy        
+    ## weight         Weight       
+    ## no_weight      No Weight    
+    ## treat2         Treat 2
+
+    ## 
+
+    ## # A tibble: 501 x 9
+    ##    gender   age sugar_factor treat  happiness happy weight no_weight treat2
+    ##    <chr>  <int>        <dbl> <chr>  <chr>     <lgl>  <dbl> <lgl>     <chr> 
+    ##  1 male      80       0.894  ice c… happy     TRUE   1.38  TRUE      pizza 
+    ##  2 female    60       0.834  candy  happy     TRUE   0.902 TRUE      pizza 
+    ##  3 male       8       0.222  candy  happy     TRUE   1.40  TRUE      ice c…
+    ##  4 female    61       0.666  candy  happy     TRUE   1.38  TRUE      pizza 
+    ##  5 male      53       0.586  candy  happy     TRUE   1.12  TRUE      ice c…
+    ##  6 female    73       0.793  ice c… happy     TRUE   0.538 TRUE      candy 
+    ##  7 female     6       0.358  candy  happy     TRUE   0.721 TRUE      ice c…
+    ##  8 male      26       0.401  ice c… happy     TRUE   0.912 TRUE      candy 
+    ##  9 male      63       0.206  candy  happy     TRUE   0.998 TRUE      candy 
+    ## 10 male      76       0.0423 candy  happy     TRUE   1.72  TRUE      candy 
+    ## # … with 491 more rows
+
+``` r
+# non-defaults
+mylabels <- list(
+  gender           = "Sex",
+  sugar_factor     = "Sugar Amount",
+  treat2           = "Alternative Treatment"
+)
+
+example_dat %>% 
+  dplyr::rename(`sugar   factor` = sugar_factor) %>% 
+  preprocess(clean_names = TRUE, convert_to_logical = FALSE, var_labels = mylabels)
+```
+
+    ## [Step 1]: Converting binary variables to logical variables
+    ## SKIPPED
+
+    ## 
+
+    ## [Step 2]: Cleaning variable names
+    ## The following variable names were cleaned with the mapping:
+    ## 
+    ## Original Name    Cleaned Name 
+    ## ---------------  -------------
+    ## sugar   factor   sugar_factor
+
+    ## 
+
+    ## Argument `label` must be of same length as numbers of columns in `x`.
+
+    ## [Step 3]: Adding variable labels
+    ## The following variable labels were specified with the mapping:
+    ## 
+    ## Variable       Label 
+    ## -------------  ------
+    ## gender               
+    ## age                  
+    ## sugar_factor         
+    ## treat                
+    ## happiness            
+    ## happy                
+    ## weight               
+    ## no_weight            
+    ## treat2
```

Comment:
Doesn't look to currently work if specifying your own labels

Reply:
"## Argument `label` must be of same length as numbers of columns in `x`."

Reply:
forgot to reknit